### PR TITLE
Adding option to testkube-api-server deployment to specify hostNetwor…

### PR DESCRIPTION
**Pull request by igaskin**
## Pull request description
Adding option to testkube-api-server deployment to specify hostNetwork and dnsPolicy 

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]
looks good. @ypoplavs please confirm

File: charts/testkube-api/values.yaml [hunk 1]
```diff
@@ -177,6 +177,12 @@ affinity: {}
 ## Specify priorityClassName for Testkube API Deployment
 priorityClassName: ""
 
+## Specify hostNetwork for Testkube API Deployment
+hostNetwork: ""
```
Comment (ypoplavs) [inline charts/testkube-api/values.yaml:181]: Could you please duplicate the values into `testkube/values.yaml` file under `testkube-api` section and run `helm-docs` to update README.md?

File: charts/testkube-api/values.yaml [hunk 2]
```diff
@@ -177,6 +177,12 @@ affinity: {}
...
+
+## Specify dnsPolicy for Testkube API Deployment
+dnsPolicy: ""
```
Comment (ypoplavs) [inline charts/testkube-api/values.yaml:184]: the same as above

File: charts/testkube-api/values.yaml (hunk 1 above)
Comment (igaskin) [reply charts/testkube-api/values.yaml:181]: sure thing, done.

File: charts/testkube-api/values.yaml (hunk 2 above)
Comment (igaskin) [reply charts/testkube-api/values.yaml:184]: sure thing, done.

**Comment by igaskin** [comment]
apologies I let this PR go stale. I just fixed the merge conflicts, can I get a re-review and possibly a merge?

**Review by ypoplavs — APPROVED** [review]

**Review by ypoplavs — APPROVED** [review]

**Pull request merged**